airbyte-cdk/python/airbyte_cdk/sources/declarative/interpolation/interpolated_boolean.py

```diff
@@ -2,23 +2,41 @@
 # Copyright (c) 2022 Airbyte, Inc., all rights reserved.
 #
 
+from typing import Any, Final, List
+
 from airbyte_cdk.sources.declarative.interpolation.jinja import JinjaInterpolation
+from airbyte_cdk.sources.declarative.types import Config
 
-false_values = ["False", "false", "{}", "[]", "()", "", "0", "0.0", "False", "false", {}, False, [], (), set()]
+FALSE_VALUES: Final[List[Any]] = ["False", "false", "{}", "[]", "()", "", "0", "0.0", "False", "false", {}, False, [], (), set()]
 
 
 class InterpolatedBoolean:
-    def __init__(self, condition):
+    """
```

Comment:
```suggestion
    f"""
```

Reply:
@sherifnada what's the benefit of making this a f-string?

Reply:
it goes with the suggestion below of showing the actual values